Automattic-Tracks-iOSTests/Tests/SentryExtensionTests.swift

```diff
@@ -0,0 +1,23 @@
+import XCTest
+@testable import AutomatticTracks
+import Sentry
+
+class SentryExtensionTests: XCTestCase {
+
+    override func setUpWithError() throws {
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func testThatEventInitializationReturnsEnglishErrorDescription() {
```

Comment:
```suggestion
    // Note: for this test to be useful, it needs to be run as part of a Test Plan which runs the test suite
    // in languages other than English. See TestPlan.xctestplan
    func testThatEventInitializationReturnsEnglishErrorDescription() {
```

Just because, even if a test plan is present, it's possible to run the tests (e.g. via command line or fastlane) without using the test plan explicitly and instead using the default config; and it's not always obvious / very visible that a Test Plan exists in the project and is used, so I think it can be useful to highlight that it does here 😉

Reply:
Yes, though you'd have to go out of your way to do so – the default scheme is set to use the test plan, so you'd need to override the defaults to make that happen. Even so, I applied this suggestion because it never hurts to document things better 😁